makefile

```diff
@@ -90,3 +90,12 @@ mkdir-output:
 .PHONY: clean
 clean:
 	$(RM) $(OUTPUT_DIR)
+	
+# run checks
+checks: dna; \
+	cp $(OUTPUT_DIR)/${JARFILE} $(R_DIR)/inst/extdata/; \
+	make rDNA; \
+	rm $(R_DIR)/inst/extdata/${JARFILE}; \
```

Comment:
Why are you creating the file and then removing it again? I don't think it's needed for making rDNA? Not sure I understand what's going on here.

Reply:
The jar file is moved to the `/inst/extdata/` folder before packaging the `tar.gz` file (so it is in the tarball). Then the jar is removed from the `rDNA` folder (but not the tarball). The only reason for this is so that the checks run with the newest version of the jar. On travis we are basically doing the same. If you want to run the checks with the latest release version you can simply run `make rDNA; R CMD check --as-cran ./*.tar.gz` instead.

Reply:
Ah, okay. Makes sense. Thx.